cycledash/static/css/examine.css

```diff
@@ -191,6 +191,9 @@ body {
   border: none;
   font-size: 13px;
   white-space: nowrap;
+  max-width: 95px;
+  text-overflow: ellipsis;
+  overflow-x: hidden;
```

Comment:
Then the overflow is never hidden, so we don't ever see the ellipsis. (http://cl.ly/image/2A1p3m052d2o)